### PR TITLE
Add Drone CI scripts

### PR DESCRIPTION
**Pull request by giordano**
If someone is willing to activate this repository in https://cloud.drone.io/, it'll enable running tests also on ARM architecture.  See https://discourse.julialang.org/t/testing-on-arm-with-drone-ci/25871

### CODE REVIEW

**Comment by ararslan** [comment]
I've activated Drone CI for this repo. I think the PR will need to be closed and reopened for it to trigger a build.

**Comment by ararslan** [comment]
This is so cool!!

File: .drone.jsonnet [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	    image: "julia:"+version,
+	    commands: [
+		"julia --project=. --check-bounds=yes --color=yes -e 'using InteractiveUtils; versioninfo(verbose=true); using Pkg; Pkg.build(); Pkg.test(coverage=true)'"
+	    ]
+	}
+    ]
+};
+
+[
+    # Pipeline("linux", "arm",   "1.0"),
```
Comment (ararslan) [inline .drone.jsonnet]: @giordano, why was this commented out? Is this something we could try enabling?

Comment (giordano) [reply .drone.jsonnet]: When this is fixed: https://github.com/JuliaMath/FFTW.jl/issues/104 🙂

Comment (giordano) [reply .drone.jsonnet]: #184 :slightly_smiling_face:

**Pull request merged**